src/appleseed.shaders/src/appleseed/as_metal.osl

```diff
@@ -62,6 +62,15 @@ shader as_metal
         string help = "Reflectance at grazing incidence.",
         int as_max_param_id = 2
     ]],
+    float in_edge_reflectance_weight = 0.0
```

Comment:
Shouldn't the default be 1?

Reply:
No. Should be 0 for compatibility with the metal roughness PBR workflow.